src/states_screens/edit_gp_screen.cpp

```diff
@@ -281,7 +281,8 @@ void EditGPScreen::setModified(const bool modified)
 
     LabelWidget* header = getWidget<LabelWidget>("title");
     assert(header != NULL);
-    header->setText(m_gp->getName() + (modified ? L" (+)" : L""), true);
+    //I18N: Indicate that the grand prix is modified and not saved
+    header->setText(modified ? _(L"%s (+)", m_gp->getName()) : L"", true);
```

Comment:
This makes "%s (+)" translatable, do we really need that to be translatable?
In any case we have a string freeze so we can't introduce new strings at this stage